src/shared/constants.ts

```
@@ -291,14 +291,15 @@
-    { id: "Mv8AjrYZCBkdsmDHNwcB", name: "Ishibashi" },
-    { id: "8EkOjt4xTPGMclNlh1pk", name: "Morioki" },
-    { id: "j210dv0vWm7fCknyQpbA", name: "Hinata" },
-    { id: "QEj0heL4nQHjaGrihlr0", name: "Steven Casteel" },
-    { id: "l39JidvAMB3s85XyNSRd", name: "Sayuri" },
+    { id: "3JDquces8E8bkmvbh6Bc", name: "Otan", key: "3JDquces8E8bkmvbh6Bc" },
+    { id: "c6SfcYrb2t09NHXiT80T", name: "Janathan", key: "c6SfcYrb2t09NHXiT80T" },
+    { id: "Mv8AjrYZCBkdsmDHNwcB", name: "Ishibashi", key: "Mv8AjrYZCBkdsmDHNwcB" },
+    { id: "8EkOjt4xTPGMclNlh1pk", name: "Morioki", key: "8EkOjt4xTPGMclNlh1pk" },
+    { id: "j210dv0vWm7fCknyQpbA", name: "Hinata", key: "j210dv0vWm7fCknyQpbA" },
+    { id: "QEj0heL4nQHjaGrihlr0", name: "Steven Casteel", key: "QEj0heL4nQHjaGrihlr0" },
+    { id: "l39JidvAMB3s85XyNSRd", name: "Sayuri", key: "l39JidvAMB3s85XyNSRd" }
```

ESLint

[error] Insert `,`

(prettier/prettier)